karma.conf.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+module.exports = function(config) {
+  config.set({
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha', 'chai'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'tests/*.spec.js'
+    ],
+
+    // list of files to exclude
+    exclude: ['bin/*'],
+    preprocessors: {
+      'tests/*.js': ['webpack', 'sourcemap']
+    },
+
+    webpack: {
+      //mode: 'production',
+      mode: 'development',
+      devtool: 'inline-source-map',
+      module: {
+        rules: [
+          {
+            test: /\.js$/,
+            exclude: [
+              /bin/,
+              /node_modules\/(?!jsonld|crypto-ld)/
+            ],
+            use: {
+              loader: 'babel-loader',
+              options: {
+                presets: ['@babel/preset-env'],
+                plugins: [
+                  '@babel/plugin-transform-modules-commonjs',
+                  '@babel/plugin-transform-runtime',
+                  '@babel/plugin-proposal-object-rest-spread'
+                ]
+              }
+            }
+          }
+        ]
+      },
+      node: {
+        Buffer: false,
+        process: false,
+        crypto: false,
+        setImmediate: false
+      },
+      externals: {
+        'bitcore-message': '\'bitcore-message\''
```

Comment:
I'm guessing this file was copied from elsewhere, is this bitcore-message thing needed here?

Reply:
it was copied for sure and no bitcore is not used here directly, but:

```
ls node_modules/bitcore-message
bower.json       dist  gulpfile.js  lib           README.md
CONTRIBUTING.md  docs  index.js     package.json  test
// it is there
```

does show tons of results so some of the signature libraries still require it.
Removing it results in non-fatal karma errors:

```
ERROR in ./node_modules/bitcore-lib/lib/crypto/random.js
Module not found: Error: Can't resolve 'crypto' in '/work/ajones/digital_bazaar/http-signature-zcap-verify/node_modules/bitcore-lib/lib/crypto'
 @ ./node_modules/bitcore-lib/lib/crypto/random.js 15:15-32
 @ ./node_modules/bitcore-lib/index.js
 @ ./node_modules/bitcore-message/index.js
 @ ./node_modules/jsonld-signatures/lib/suites/EcdsaKoblitzSignature2016.js
 @ ./node_modules/jsonld-signatures/lib/suites.js
 @ ./node_modules/jsonld-signatures/lib/jsonld-signatures.js
 @ ./tests/10-verify.spec.js
```

so thanks for the check as it does appear it is still required.